src/Components/CheckOut/CheckOut.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { useState } from "react";
+import { IoIosArrowBack } from "react-icons/io";
+import { useNavigate } from "react-router-dom";
+import { CHECKOUT_TEXTS } from "../../utils/constants/constants";
+import {
+  IBasketProduct,
+  IBasketText,
+  ICatchError,
+} from "../../utils/interface";
+import useGetText from "../basket/utils/useGetText";
+import CheckOutCart from "./components/CheckOutCart";
+import { SendCheckoutOrder } from "./utils.tsx/SendCheckoutOrder";
+
+const CheckOut = ({
+  checkoutBasket,
+  setCheckoutBasket,
+}: {
+  checkoutBasket: IBasketProduct[];
+  setCheckoutBasket: React.Dispatch<React.SetStateAction<IBasketProduct[]>>;
+}) => {
+  const [firstName, setFirstName] = useState<string>("");
+  const [lastName, setLastName] = useState<string>("");
+  const [company, setCompany] = useState<string>("");
+  const [userEmail, setUserEmail] = useState<string>("");
+  const [phoneNumber, setPhoneNumber] = useState<string>("");
+  const [userAddress, setUserAddress] = useState<string>("");
+  const [postAddress, setPostAddress] = useState<string>("");
+  const [sendError, setSendError] = useState<boolean>(false);
+  const [chekOutSendError, setCheckOutSendError] = useState<ICatchError>();
+  const [sendLoading, setSendLoading] = useState<boolean>(false);
+
+  const [checkoutText, setCheckoutText] = useState<IBasketText[] | undefined>();
+  useGetText(setCheckoutText, CHECKOUT_TEXTS);
+  const deleveriPrice = checkoutText && checkoutText[29].description;
```

Comment:
wait for Karen to apply object based response scheme. after that change all indexes